.gitignore

```diff
@@ -11,6 +11,7 @@ requirements.txt
 build
 .cython_build
 .hypothesis
+.ropeproject
```

Comment:
What's this? We should avoid any autodoc thing that isn't standardized across the project I think, to avoid spurious refactors of the code when different developers work on the code